client-src/elements/form-field-specs.ts

```diff
@@ -2125,6 +2125,14 @@ export const ALL_FIELDS: Record<string, Field> = {
     take action to continue using some existing functionaity.`,
   },
 
+  confidential: {
+    type: 'checkbox',
+    label: 'Confidential',
+    initial: false,
+    help_text: html `This is a confidential feature that should not be publicly
```

Comment:
It would be more clear if this explained who could view the feature.  E.g., "@google.com accounts, @chromium.org accounts, the feature owners, and editors."